.github/workflows/php-test-plugins.yml

```diff
@@ -43,11 +43,15 @@ jobs:
       matrix:
         php: ['8.2', '8.1', '8.0', '7.4', '7.3', '7.2']
         wp: [ 'latest' ]
+        coverage: [false]
         include:
           - php: '7.4'
             wp: '6.5'
           - php: '8.3'
             wp: 'trunk'
+          - php: '8.2'
+            wp: 'latest'
+            coverage: true
```

Comment:
I wonder, given we include this as an extra scenario, should we maybe remove `8.2` from the regular `php` list above? Otherwise we'd be running the exact same test (WP latest, PHP 8.2) twice, only with and without code coverage, which shouldn't change the outcome of the tests.

If there's any benefit to keeping both (which would only be because of the implications that running code coverage could have), then I think the better choice would be to implement a separate workflow entirely.

Reply:
Yes make sense, removing the 8.2 version from PHP list.